### ISSUE TITLE
Exception: wrapped C/C++ object of type TyphosAlarmCircle has been deleted

### ISSUE THREAD

**Issue by klauer** [pcdshub/typhos#470]
## Expected Behavior
No error.

## Current Behavior
Repetitive errors being caught by logging system (hundreds over a month):
```
Traceback (most recent call last):
  File "/cds/group/pcds/pyps/conda/py39/envs/dev/bin/lucid", line 10, in <module>
    sys.exit(main())
  File "/cds/group/pcds/pyps/apps/dev/pythonpath/lucid/launcher.py", line 246, in main
    launch(**kwargs)
  File "/cds/group/pcds/pyps/apps/dev/pythonpath/lucid/launcher.py", line 240, in launch
    app.exec_()
  File "/cds/group/pcds/pyps/apps/dev/pythonpath/typhos/alarm.py", line 301, in update_severity
    self.update_current_alarm()
  File "/cds/group/pcds/pyps/apps/dev/pythonpath/typhos/alarm.py", line 316, in update_current_alarm
    self.alarm_changed.emit(new_alarm)
RuntimeError: wrapped C/C++ object of type TyphosAlarmCircle has been deleted
```

I have not tracked this to the source just yet, whether it's due to LUCID's usage or the typhos internals.

## Possible Solution
Track down issue here or in LUCID where this occurs.

## Steps to Reproduce (for bugs)
Unclear at this point; no direct user reports.

## Context
Seen in log messages from LUCID home screens.

## Your Environment
Seen messages from pcds-4.2.0, pcds-dev, etc. I believe it's still an issue.

**Comment by klauer** [comment]
Jyoti just reported this (as the first user who's seen it _and_ let the PD team know). She noted that RIX scientists have often complained about this. I think we should boost its priority.

**Comment by jyotiphy** [comment]
@klauer I tried couple of times what I was doing earlier but not able to reproduce the error message.

**Comment by jyotiphy** [comment]
I noticed these error(s) in control-room and scientists have complained about these few times...see screenshot from the elog which I just took. Not sure how to reproduce these though. 

https://pswww.slac.stanford.edu/lgbk/lgbk/rixly5620/ws/attachment?entry_id=61f040167aaa2edf2537287a&attachment_id=61f040157aaa2edf25372877&prefer_preview=False 

https://pswww.slac.stanford.edu/lgbk/lgbk/rixly5620/ws/attachment?entry_id=61f04084c31c44e4a6f87b4f&attachment_id=61f04084c31c44e4a6f87b4c&prefer_preview=False

**Comment by klauer** [comment]
Thanks, @jyotiphy!

**Comment by ZLLentz** [comment]
Maarten reports that this happens in TMO and confuses the scientists

**Comment by ZLLentz** [comment]
It took me a while, but I managed to reproduce this.

This happens even in pure typhos, not just in the context of lucid. The steps to reproduce are:

1. Open a device screen that has a "TyphosAlarmCircle" in it, such as any of the detailed positioner screens.
2. Switch to an alternate view of this screen that has no such circle in it, such as the "engineering" screen
3. Cause any change to the alarm state (e.g., toggle off or on your test soft ioc)

Now that I've reproduced it in a way where I can trigger the error, we are one step closer to a resolution.
